plugins/modules/source_control/github/github_repo.py

```diff
@@ -42,16 +42,16 @@
   description:
     description:
     - Description for the repository.
+    - Defaults to empty when creating a new repository.
```

Comment:
It also defaults to empty if `force_defaults=true`, which is the default in this module. You should mention that here.